src/ansys_sphinx_theme/theme/ansys_sphinx_theme/theme.conf

```diff
@@ -12,3 +12,4 @@ hidden_icons =
 additional_breadcrumbs =
 use_edit_page_button = True
 switcher =
+use_meilisearch=
```

Comment:
Are we going to set some default values here?

Reply:
Since we are not forcing anybody to use meilisearch , i think we can leave as an empty value (if using MeiliSearch functionality, else pydata functionality) , and may add the defaults, once the libs starts using it.

Reply:
Sounds good to me then. Thanks @Revathyvenugopal162